requirements.txt

```diff
@@ -1,3 +1,5 @@
-netCDF4==1.6.4
```

Comment:
https://stackoverflow.com/questions/77507580/userwarning-figurecanvasagg-is-non-interactive-and-thus-cannot-be-shown-plt-sh

fehlt noch